.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,3 +1,9 @@
+---
+name: Bug Report
+about: Something not up to standard? Let us know!
+labels: 'bug, needs triage'
+---
+
 Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new features, add an issue in our [Request Feature Request](https://github.com/reactioncommerce/reaction-feature-requests) repository.
```

Comment:
I'd suggest we include the checklist and initial instructions in comment blocks as well. These frequently show up in new issues and I don't think they need to? 

There's an argument for leaving in the checklist, but I think the main goal is that people read it and consider whether they need to create an issue now or if they need to make sure they are on the right version and search for other issues there and I don't think that's any more likely with a visible checklist.

Reply:
the checklist for the prereqs is now included in the first comment block